http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -266,9 +266,13 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
     optionalHeaderValueByType(Referer) flatMap {
       case Some(referer) => {
         if (refererConfig.validHosts.contains(referer.uri.authority.toString())) pass
-        else failWith(AuthenticationError())
+        else {
+          logger.info(s"${referer} is not allowed")
```

Comment:
these logs are only accessible by us internally correct?

Reply:
yea

Reply:
maybe we could change this to `logger.info(s"${referer} is not a valid Referer")` so that it's easier to search the logs for the string "Referer" and find all referer logs with the same grep

Reply:
+1, would be good to include the string "Referer" in this log msg